### ISSUE TITLE
AlgoliaSearchError: Index scanpy does not exist

### ISSUE THREAD

**Issue by ivirshup** [algolia/docsearch-configs#4306]
# Do you want to request a **feature** or report a **bug**?

### If it is a DocSearch index issue, what is the related `index_name` ?

`index_name`= scanpy

## What is the **current** behaviour?

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

Code search doesn't seem to be working, e.g. nothing happens as I type a query. At each keystroke, I get an error in the console:

```
Failed to load resource: https://bh4d9od16a-dsn.algolia.net/1/indexes/*/queries?x-algolia-agent=Algolia%20for%20vanilla%20JavaScript%20(lite)%203.30.0%3Bdocsearch.js%202.6.3&x-algolia-application-id=BH4D9OD16A&x-algolia-api-key=fa4304eb95d2134997e3729553a674b2
 the server responded with a status of 404 (Not Found)
```

This seems similar to https://github.com/algolia/docsearch-configs/issues/4277

## What is the **expected** behaviour?

I would expect searching to work, and yield results.

#### What have you tried to solve it?

I've mostly googled around for similar issues. Since this was working previously, and I don't think we touched anything around this, I'm not expecting this to be on our side.

#### Any quick clues?

I'm not sure how long it's been broken. I only noticed when we were making the release that had this change and noticed the builds didn't have search working.

**Comment by shortcuts** [comment]
Hi, thanks for reporting. 

Please see: [this comment](https://github.com/algolia/docsearch-configs/issues/4277#issuecomment-870001301) where I explain why you index doesn't exists anymore. 

TLDR: Your config is probably outdated and needs to be updated, please make it compliant with your website and open a Pull Request

Thanks!

**Comment by ivirshup** [comment]
Thanks for the quick response!

Would you mind giving me some pointers on this? I'm not too sure which parts of my config could be out of date, or how I could tell if they were.

**Comment by shortcuts** [comment]
I'd suggest you to add the direct link to the website `https://scanpy.readthedocs.io/` at the end of the `start_urls` array (since `https://scanpy.readthedocs.io/en/` leads to a 404), and check if there's any selectors that don't match anymore!

**Comment by sandboxplayer** [comment]
You are using the appId of Apache Cordova (bh4d9od16a) , you should  replace it with your own.

**Comment by ivirshup** [comment]
@sandboxplayer I'm not sure what you mean by replace the appId, since I don't believe we specify it anywhere. I believe that is set on the algolia side.

@shortcuts, I've opened a PR with your suggestion, and a second change I thought might be the issue.

**Comment by shortcuts** [comment]
> You are using the appId of Apache Cordova (bh4d9od16a) , you should  replace it with your own.

This app id is the default DocSearch one and is shared with all the DocSearch users
